### PR TITLE
feat: added NativeScript to Hacktoberfest swag list

### PR DESCRIPTION
**Pull request by akk312000**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [ ] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [ ] I'm not adding a company from the blocklist
3. [ ] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by akk312000** [comment]
Solves Issue #649 @crweiner .

File: docs/list.md [hunk 1]
```diff
@@ -330,6 +330,21 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
 
+#### **NativeScript**
+
+- **Swag**: Exclusive NativeScript Swags
+- **Requirements**:
+    - Create a StackBlitz of anything fun or neat about [NativeScript](https://preview.nativescript.org){:target="\_blank"}.
+    - Contribute a PR for an issue in a participating NativeScript repository.
+    - Share a gist of neat NativeScript code you find useful.
+    - Create a NativeScript plugin.
+    - Tweet about it and mention @NativeScript.
```
Comment (crweiner) [inline docs/list.md:341]: Can you please have the @Nativescript part link to their Twitter account?

File: docs/list.md [hunk 2]
```diff
@@ -330,6 +330,21 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+    - DM us your address to get free swag.
```
Comment (crweiner) [inline docs/list.md:342]: Please don't DM _me_ your address! 😆 Please make this clear that NativeScript is looking for a DM.

**Review by crweiner — CHANGES_REQUESTED** [review]
Hey @akk312000! 👋 Thanks for your PR! I've fixed the trailing whitespace flags from Code Climate for you (sometimes an IDE or your computer adds them as you type) and would like some small tweaks as pointed out below, please. Thank you! 😄

**Comment by akk312000** [comment]
@crweiner  Updated the PR, please have a look, Thanks.

**Review by crweiner — APPROVED** [review]
Thank you for the updates, @akk312000! I appreciate it! Everything looks great now, so let's get this on the Swag List! 

Thank you for your PR and have a great Hacktoberfest!

thank you

**Comment by akk312000** [comment]
@crweiner A big Thanks to you for allowing me to contribute to it :). 😁

**Pull request merged**